climada/util/plot.py

```diff
@@ -328,9 +328,21 @@ def geo_im_from_array(array_sub, coord, var_name, title,
     if not isinstance(axes, np.ndarray):
         axes_iter = np.array([[axes]])
 
-    if 'cmap' not in kwargs:
-        kwargs['cmap'] = CMAP_RASTER
-
+    # prepare colormap
+    cmap = plt.get_cmap(kwargs.pop("cmap", CMAP_RASTER))
+    cmap.set_bad("gainsboro")  # For NaNs and infs
+    cmap.set_under("white", alpha=0)  # For values below vmin
+
+    # prepare boundary values
+    if "norm" in kwargs:
+        min_value = kwargs["norm"].vmin
+        vmin = None  # We will pass norm
+        del kwargs["vmin"], kwargs["vmax"]
```

Comment:
Matplotlib will throw an error if vmin/vmax *and* norm are passed. We should not handle that silently.
```suggestion
        del kwargs["vmin"], kwargs["vmax"]
```

Reply:
This is why I am deleting these entries here. Earlier in the function (lines 312-315) we add the vmin/vmax kwargs, but as you say if we also input norm this will result in an error. How do you think we should handle this?

Reply:
Woops, I was not aware of this 😅 I have to say these four lines confuse me a bit, because it should be the default behavior of any matplotlib norm to use `nanmin` and `nanmax` of the inserted values as `vmin` and `vmax`, respectively. So this looks like a no-op to me. Can you check if we can simply remove them?

Reply:
I believe the reason for these lines is that we want to have that same colorbar for all the subplots, not a colorbar that changes its min/max bounds depending on the subplot? At least this is what happens if I comment about the lines. I assume there could be a more elegant way to code this, do you have a suggestion?